### PR TITLE
Flash tn t vis

### PR DESCRIPTION
**Pull request by Ayesha-Feroz**
FDR plot added

### CODE REVIEW

File: src/components/ui/ComponentsLayout.vue [hunk 1]
```diff
@@ -44,3 +47,11 @@ export default defineComponent({
   flex-direction: column;
 }
 </style>
+
+<FDRPlotly
```
Comment (t0mdavid-m) [inline src/components/ui/ComponentsLayout.vue:51]: This should not be necessary and must be removed. Can you try if you can get the plot to be displayed without it? You can also open a PR to my FLASHViewer fork. Then we could troubleshoot it together.

File: src/components/ui/ComponentsRow.vue [hunk 2]
```diff
@@ -54,6 +54,14 @@
         :index="componentIndex(index)"
       />
       <FLASHQuantView v-else-if="component.componentArgs.componentName === 'FLASHQuantView'" />
+
+      <!-- New block added for FDRPlotly component -->
```
Comment (t0mdavid-m) [inline src/components/ui/ComponentsRow.vue:58]: The comments around the new block are not necessary.

**Review by t0mdavid-m — CHANGES_REQUESTED** [review]
Looks good to me as a first step. The plot should work without hard coding it in the layout though. Do you want to add the currently existing bar plot as well or is not relevant anymore?

I think this would be a good resource on how to add it (as a separate component):
https://plotly.com/javascript/bar-charts/

File: src/components/ui/ComponentsLayout.vue (hunk 1 above)
Comment (Ayesha-Feroz) [reply src/components/ui/ComponentsLayout.vue:51]: @t0mdavid-m I think a single plot would be sufficient—either the ECDF plot or the density histogram of target and decoy Qscores, with counts on the y-axis. What do you think?

Comment (Ayesha-Feroz) [reply src/components/ui/ComponentsLayout.vue:51]: Screenshot 2024-10-29 at 09 03 46

Comment (t0mdavid-m) [reply src/components/ui/ComponentsLayout.vue:51]: Up to you! We could always implement one plot and add the other plot later if necessary.

Did you try removing the hardcoded section in `ComponentsLayout.vue`?

Comment (Ayesha-Feroz) [reply src/components/ui/ComponentsLayout.vue:51]: i will do it after lunch. and keep you updated if its done

Comment (Ayesha-Feroz) [reply src/components/ui/ComponentsLayout.vue:51]: @t0mdavid-m the 

> This should not be necessary and must be removed. Can you try if you can get the plot to be displayed without it? You can also open a PR to my FLASHViewer fork. Then we could troubleshoot it together.

your suggestion is incorporated and the code block is removed and without it its also showing the fdr plot.

**Pull request merged**